frontend-react/package.json

```diff
@@ -65,15 +65,21 @@
         "build-base-dev": "yarn compile-scss-dev && node scripts/build.js && yarn copy-404page",
         "start-js": "node scripts/start.js",
         "copy-404page": "cp build/index.html build/404.html",
-        "compile-scss-prod": "sass --load-path=./node_modules/@uswds/uswds/packages --no-source-map --style=compressed --quiet src/global.scss:src/content/generated/global.out.css",
-        "compile-scss-dev": "sass --load-path=./node_modules/@uswds/uswds/packages --embed-source-map --quiet-deps src/global.scss:src/content/generated/global.out.css",
+        "compile-scss-prod": "npm-run-all compile-global-scss-prod compile-content-scss-prod",
+        "compile-scss-dev": "npm-run-all compile-global-scss-dev compile-content-scss-dev",
+        "compile-global-scss-prod": "sass --load-path=./node_modules/@uswds/uswds/packages --no-source-map --style=compressed --quiet src/global.scss:src/content/generated/global.out.css",
+        "compile-global-scss-dev": "sass --load-path=./node_modules/@uswds/uswds/packages --embed-source-map --quiet-deps src/global.scss:src/content/generated/global.out.css",
+        "compile-content-scss-prod": "sass --load-path=./node_modules/@uswds/uswds/packages --no-source-map --style=compressed --quiet src/content.scss:src/content/generated/content.out.css",
+        "compile-content-scss-dev": "sass --load-path=./node_modules/@uswds/uswds/packages --embed-source-map --quiet-deps src/content.scss:src/content/generated/content.out.css",
         "cypress:local:run": "cross-env CYPRESS_ENV=development yarn cypress run",
         "cypress:local:open": "cross-env CYPRESS_ENV=development yarn cypress open",
         "cypress:staging:run": "echo 'not implemented yet'",
         "cypress:staging:open": "echo 'not implemented yet'",
         "cypress:production:run": "echo 'not implemented yet'",
         "cypress:production:open": "echo 'not implemented yet'",
-        "watch-scss": "yarn compile-scss-dev && sass --load-path=./node_modules/@uswds/uswds/packages --embed-source-map --quiet-deps -w src/global.scss:src/content/generated/global.out.css",
+        "watch-scss": "npm-run-all -p watch-global-scss watch-content-scss",
```

Comment:
Just curious, why are we breaking this out like this?

Reply:
Can you elaborate on your question?

Reply:
like, why do we want to run `watch-global-scss` and `watch-content-scss` concurrently with these changes.

Reply:
Due to now creating two different css files, watching for changes (when running local dev) requires running both processes in parallel. Any changes to any files (except the entry scss file and their specific settings override file) affects both css outputs.